docs/operator-manual/notifications/index.md

```diff
@@ -25,22 +25,22 @@ kind: Secret
 metadata:
   name: argocd-notifications-secret
 stringData:
-  email-username: $EMAIL_USER
-  email-password: $PASSWORD
+  email-username: <email_address>
+  email-password: <email_password>
```

Comment:
If you prefer this over env vars, I'd recommend removing the `export` lines above.